docs/developer/languages/go/components.mdx

```diff
@@ -34,9 +34,8 @@ In this walkthrough, we will create an HTTP server from scratch using the [Go co
 
 * [Go](https://go.dev/doc/install) 1.23.0+
 * [TinyGo](https://tinygo.org/getting-started/install/) 0.33.0+
-* [wasmCloud Shell (`wash`)](https://wasmcloud.com/docs/installation) CLI 0.32.1+ for building and deploying components
-  {/* TODO: Remove after https://github.com/wasmCloud/wasmCloud/pull/3270 is released (est. wash-cli@v0.36.0) */}
-* [`wasm-pkg-tools`](https://github.com/bytecodealliance/wasm-pkg-tools) 0.7.4+ for WIT dependency management
+* [wasmCloud Shell (`wash`)](https://wasmcloud.com/docs/installation) CLI 0.36.0+ for building and deploying components
+* [`wasm-tools`](https://github.com/bytecodealliance/wasm-tools#installation) for Go binding generation
```

Comment:
IIRC we need *both* `wasm-tools` and `wit-bindgen-cli`, right now?

Unless things have changed, we [install both in CI](https://github.com/wasmCloud/wasmCloud/blob/8c96324fc0b28a475073974b23451e19736070ae/.github/actions/wasm-lang-toolchain-setup/action.yml#L35)